akka-sample-sharding-scala/src/main/resources/application.conf

```diff
@@ -1,5 +1,7 @@
 akka {
   loglevel = INFO
+  loggers = ["akka.event.slf4j.Slf4jLogger"]
+  logging-filter = "akka.event.slf4j.Slf4jLoggingFilter"
```

Comment:
those two are enabled automatically when using akka-actor-typed, and I'd recommend setting loglevel = DEBUG here and control level from logback.xml instead